packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -219,11 +219,7 @@ const useTransactionHistoryWithoutStatuses = (address: Address | undefined) => {
         return undefined
       }
       // EOA
-      // fetch all for testnet mode, do not fetch testnets if testnet mode disabled
-      if (isTestnetMode) {
-        return 'all'
-      }
-      return isNetwork(chainPair.parentChainId).isTestnet ? undefined : 'all'
+      isNetwork(chainPair.parentChainId).isTestnet === isTestnetMode
```

Comment:
This expression does nothing, it's not returned

Reply:
updated